src/pybind/mgr/dashboard/frontend/e2e/block/iscsi.po.ts

```diff
@@ -0,0 +1,7 @@
+import { browser } from 'protractor';
+
+export class iSCSIPage {
```

Comment:
Classes need to be in Pascal case. We use the following format in the iSCSI component:
```suggestion
export class IscsiPage {
```